components/style/properties/properties.mako.rs

```diff
@@ -1145,7 +1145,7 @@ impl ComputedValues {
     #[inline]
     pub fn is_multicol(&self) -> bool {
         let style = self.get_column();
-        style.column_count.0.is_some() || style.column_width.0.is_some()
+        style.column_count.0.is_some() || style.column_width.is_first()
```

Comment:
We should do something like,

``` rust
match (style.column_count, style.column_width) {
    (Some(ref _count), Either::First(ref _length)) => true,
    _ => false,
}
```

... so that it'll be self-explanatory :)